recipes/nvjitlink/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nvjitlink" %}
+{% set version = "0.1.0.dev1" %}
```

Comment:
```suggestion
{% set version = "0.1.0" %}
```

Reply:
This should be updated I agree but I'd like to get https://github.com/rapidsai/nvjitlink/pull/3 in otherwise I'd expect test failures here

Reply:
Well [the 0.1.0 release]( https://github.com/rapidsai/nvjitlink/releases/tag/0.1.0 ) already exists

Currently [CI fails]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=748133&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1381 ) since this isn't a real path / URL